src/everest/optimizer/everest2ropt.py

```diff
@@ -65,9 +65,18 @@ def _parse_controls(ever_controls: list[ControlConfig], ropt_config):
         ropt_config["gradient"] = {}
 
     if any(item >= 0 for item in controls.sampler_indices):
+        if any(sampler.backend is not None for sampler in controls.samplers):
+            message = (
+                "sampler.backend is deprecated. "
+                "The correct backend will be inferred by the method. "
+                "If several backends have a method named A and you want to pick "
+                "a specific backend B, put B/A in sampler.method."
+            )
+            print(message)
```

Comment:
Shouldn't this deprecation message come from `everest/config/sampler_config.py` ?